code/game/gamemodes/objective.dm

```diff
@@ -389,6 +389,10 @@ GLOBAL_LIST_INIT(potential_theft_objectives, (subtypesof(/datum/theft_objective)
 		explanation_text = "Steal [steal_target]. One was last seen in [get_location()]. "
 		if(length(O.protected_jobs))
 			explanation_text += "It may also be in the possession of the [english_list(O.protected_jobs, and_text = " or ")]."
+		if(istype(O, /datum/theft_objective/supermatter_sliver))
```

Comment:
This is ❄️ 

This should be made more OOP by making it a proc that equips the user if the objective's overridden has it defined.

`give_special_objective_gear` or something, with the default value being `return` for the ultimate parent type.

Reply:
Changed it to a special_equipment variable on the objectives, with parent being null